core/controllers/base_test.py

```diff
@@ -1423,3 +1469,407 @@ def test_that_correct_derived_class_does_not_raise_exception(self):
         with self.swap(self, 'testapp', self.mock_testapp):
             self.post_json(
                 '/correctmock', payload, expected_status_int=200)
+
+
+class SchemaValidationIntegrationTests(test_utils.GenericTestBase):
+    """Tests all the functionality of SVS(Schema-Validation-System)
+    architecture.
+    """
+
+    non_schema_handlers = payload_validator.NON_SCHEMA_HANDLERS
+    non_schema_requiring_handlers = (
+            payload_validator.NON_SCHEMA_REQUIRING_HANDLERS)
+    handlers_need_schema = []
+    handlers_to_remove = []
+    handlers_with_missing_url_schema_keys = []
+    handlers_with_missing_req_schema_keys = []
```

Comment:
Don't know what "req" is, please use the full name.

Reply:
Done